LCM/dsc/engine/ca/CAInfrastructure/WebPullClient.c

```diff
@@ -2077,23 +2077,20 @@ MI_Result MI_CALL Pull_GetModules(_Out_ MI_Uint32 * numModulesInstalled,
         DSC_LOG_INFO("Assuming python2 in WebPullClient\n");
 
 	// Look for python2
```

Comment:
Same here and please also check other places.

Reply:
Changes are pushed as per discussion